doc/developer/chips/18.rst

```diff
@@ -373,11 +392,18 @@ many NUMA-sensitive arrays turned up one that looks like this:
        }
     }
 
-Note that if either the for-stmt cannot be converted to a forall-stmt or
-``complicatedExpensiveComputation()`` really is expensive then the
-``nextLayer`` references in the kernel may not really be NUMA-sensitive.
-But for now this is is all we have seen that might be sensitive to the
-cost of array-by-array registration as described here.
+This is a start, but note that if either the for-stmt here cannot be
+converted to a forall-stmt or ``complicatedExpensiveComputation()``
+really is as expensive as its name implies then the ``nextLayer``
+references in the loop kernel may not actually be particularly
+NUMA-sensitive, in that getting them perfectly right makes the loop run
```

Comment:
Do you mean 'in that getting them perfectly right does not make the loop run any faster overall'?